hack/print-workspace-status.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright 2018 gRPC authors.
```

Comment:
I suspect this was copied from a different repo, if that is the case, then we shouldn't change this. Perhaps @liztio can provide some history here.

Reply:
Yeah, I stole this from the k8s repo. I did modify it a bit though, so not sure if that means I get to change it? Licenses are tricky.

Reply:
checked in on slack, @liztio is OK changing the license here.